server/NewBank.java

```diff
@@ -19,33 +20,45 @@ private NewBank() {
 	}
 
 	private void addTestData() {
-		Customer bhagy = new Customer();
-		bhagy.addAccount(new Account("Main", 1000.0));
-		customers.put("Bhagy", bhagy);
+		newbank.server.Customer bhagy = new newbank.server.Customer("Bhagy", "1234");
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it